data/campaigns/Northern_Rebirth/scenarios/02_02_Meeting_With_Dwarves.cfg

```diff
@@ -130,22 +130,22 @@
 
         [message]
             speaker=Hamel
-            message= _ "Oh, there are orcs enough down here, but ye’ll not likely see them; they hide from us like vermin. You see, lad, in these caves there is no more fearsome a foe than a dwarf. Not only do we move through the caves swiftly, but we know them like the backs of our hands. The orcs did indeed manage to swamp our defenses with their sheer numbers all those long years ago, but not before we surprised and slew their leader — the cursed Khazg Black-Tusk. Since then, although the orcs still foul these caves with their stench, their attacks have been uncoordinated and erratic, wi’ the orcs as likely to attack each other as they are to attack us."
+            message= _ "Oh, there are orcs enough down here, but ye’ll not likely see them; they hide from us like vermin. You see, lad, in these caves there is no more fearsome a foe than a dwarf. Not only do we move through the caves swiftly, but we know them like the backs of our hands. All those years ago, the orcs did indeed manage to swamp our defenses with their sheer numbers, but not before we surprised and slew their leader — that cursed Khazg Black-Tusk. Since then, although the orcs still foul these caves with their stench, their attacks have been uncoordinated and erratic, wi’ the orcs as likely to attack each other as they are to attack us."
```

Comment:
I prefer the original for both changes.

The subject of the previous sentence is about dwarven movement points, so starting the new sentence with "The orcs" is a clear subject change. "The orcs did indeed" manages in 4 words to change subject and say that this will be an admission of something, so it's no surprise to have it qualified later in the sentence with "long years ago"; the new version is putting the timeframe before we know what the subject is.

It's possible "the cursed Black-Tusk" vs "that cursed Black-Tusk" is an en_GB vs en_US thing, but to me "that cursed Black-Tusk" would mean that the act of slaying their old leader put a curse on Black-Tusk.

Reply:
I don't think it's a regional difference - at least not from my en_GB perspective. As a native English speaker I read the original as more suitable for a passive third-person perspective (ie external narrator), while the change seems more like a reference from a first-person speaker - which is the perspective given here. On the other hand, with the ambiguity of English, it could be read in the way that steve describes too. If that is a potential consequence of the change then perhaps it better to leave it as-is.

Reply:
Unless Hamel is crazy and has a weird announcer voice, I'd say leave it. I don't see how you see how you can think it means they cursed Black-Tusk.

I added an explanation point at the end of the sentence. However, if it's still confusing we could change the cursed to another curse word: blasted, bloody?